clients/client-shell/cmds/version/version.go

```diff
@@ -18,13 +47,69 @@ var (
 
 	// VersionNumber is a formatted string with the version information. This is
 	// filled in by `yarn release`
-	VersionNumber = "24.1.10"
+	VersionNumber = "24.1.9"
 )
 
 func init() {
 	root.Command.AddCommand(Command)
+	root.Command.AddCommand(Updcommand)
 }
 
+
 func printVersion(cmd *cobra.Command, _ []string) {
 	fmt.Fprintf(cmd.OutOrStdout(), "taskcluster version %s\n", VersionNumber)
 }
+
+func check(cmd *cobra.Command, _ []string) {
+
+    //Get response and also check for error in  from git's REST API
+
+    response, err := http.Get ("https://api.github.com/repos/taskcluster/taskcluster/releases/latest") 
+
+    if err != nil {
+        fmt.Print(err.Error())
+        
+    }
+
+    // Read the whole response body and check for any errors
+
+
+    s, err := ioutil.ReadAll(response.Body)
+    
+    if err != nil {
+        fmt.Errorf(err.Error())
+        
+    }
+
+    //Create an object for the struct to parse the json data into given structure
+    
+    R := Release{}
+    if err := json.Unmarshal([]byte(s), &R); err != nil {
+        fmt.Errorf(err.Error())
+    }
+
+    //Check if taskcluster is already up to date
+
+	if R.Name == VersionNumber {
+       
+        fmt.Println("taskcluster version  %s is already updated\n", VersionNumber)
+    
+    
+    } else {
+    fmt.Println("taskcluster curent version  %s is older than the latest %s \n", VersionNumber,R.Name)	
```

Comment:
`Println` doesn't take string formatting in this way (and the CI job is reporting this at the link below. `fmt.Printf` should be fine. 
Nitpick: "current"


 https://community-tc.services.mozilla.com/tasks/Qii0N7nbRwWnd_5TRy8_CQ/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FQii0N7nbRwWnd_5TRy8_CQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L197-198

Reply:
Done